packages/styles/forms.css

```diff
@@ -210,7 +218,7 @@ textarea.Field--has-error:focus:hover,
   text-align: left;
   color: var(--field-label-text-color);
   font-size: var(--text-size-small);
-  font-weight: var(--font-weight-bold);
+  font-weight: var(--font-weight-medium);
```

Comment:
Not directly related to this ticket, but Bobby noticed the font-weight for Field__label was off.